CHANGELOG.md

```diff
@@ -1,11 +1,4 @@
-## 5.0.0-alpha.2
-### Bugfixes
-* [BC5] Map SubscriptionOption.PricingPhases (#298) via Josh Holtz (@joshdholtz)
-* Fix product changes and add a way to test in the Subtester app
-### Other Changes
-* Added buttons to purchase all SubscriptionOptions in the Subtester app
-
-## 5.0.0-alpha.1
```

Comment:
Ideally we would leave the alphas in the changelog I think for reference. We can move the big announcement to the final `5.0.0` though.

Reply:
I just checked what we did in other hybrids and we didn't keep alpha changelogs, but in all fairness we didn't keep a real alpha changelog for those 😅 I'll add these these comments to the PR body for reference instead.